### PR TITLE
fix(data-platform): Clarify requirement of Prisma ORM

### PR DESCRIPTION
**Pull request by janpio**
PDP does not actually work with any other open source libraries.

### CODE REVIEW

File: content/800-data-platform/010-about/index.mdx [hunk 1]
```diff
@@ -1,12 +1,12 @@
 ---
 title: 'About the Prisma Data Platform'
 metaTitle: 'About the Prisma Data Platform'
-metaDescription: 'Learn about using the Prisma Data Platform. The Prisma Data Platform provides a collaborative, cloud-based environment supporting application developers who are using Prisma and other open source tools. The main features of the Platform include an online data browser, a query console, a data proxy (purpose-built for Prisma!), and integration with your Prisma schema and databases.'
+metaDescription: 'Learn about using the Prisma Data Platform. The Prisma Data Platform provides a collaborative, cloud-based environment supporting application developers who are using Prisma ORM. The main features of the Platform include an online data browser, a query console, a data proxy (purpose-built for Prisma!), and integration with your Prisma schema and databases.'
```
Comment (ruheni) [inline content/800-data-platform/010-about/index.mdx]:
```suggestion
metaDescription: 'Learn about using the Prisma Data Platform. The Prisma Data Platform provides a collaborative, cloud-based environment supporting application developers who are using Prisma ORM. The platform's main features include an online data browser, a query console, a data proxy (purpose-built for Prisma!), and integration with your Prisma schema and databases.'
```

File: content/800-data-platform/010-about/index.mdx [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 ---
 
 <TopBlock>
 
-The [Prisma Data Platform](https://cloud.prisma.io/) provides a collaborative, cloud-based environment supporting application developers who are using Prisma and other open source tools. The main features of the Platform include an [online data browser](https://www.prisma.io/blog/prisma-online-data-browser-ejgg5c8p3u4x), a query console, a [data proxy](/data-platform/data-proxy) (purpose-built for Prisma!), and integration with your Prisma schema and databases.
+The [Prisma Data Platform](https://cloud.prisma.io/) provides a collaborative, cloud-based environment supporting application developers who are using Prisma ORM. The main features of the Platform include an [online data browser](https://www.prisma.io/blog/prisma-online-data-browser-ejgg5c8p3u4x), a query console, a [data proxy](/data-platform/data-proxy) (purpose-built for Prisma!), and integration with your Prisma schema and databases.
```
Comment (ruheni) [inline content/800-data-platform/010-about/index.mdx:9]:
```suggestion
The [Prisma Data Platform](https://cloud.prisma.io/) provides a collaborative, cloud-based environment supporting application developers who are using Prisma. The main features of the Platform include an [online data browser](https://www.prisma.io/blog/prisma-online-data-browser-ejgg5c8p3u4x), a query console, a [data proxy](/data-platform/data-proxy) (purpose-built for Prisma!), and integration with your Prisma schema and databases.
```

**Comment by janpio** [comment]
What is your motivation for not explicitly calling it "Prisma ORM"?

I like the "... the platform's main ..." change - makes it sounds less scary.

**Comment by ruheni** [comment]
>What is your motivation for not explicitly calling it "Prisma ORM"?

Natually, my brain associated "Prisma" with "Prisma ORM", hence change. I think we could prefix "Prisma ORM" with the "the" article - the Prisma ORM - if we consider it a noun.

**Comment by janpio** [comment]
I would consider it a product name, for which I would not use `the`.

**Comment by janpio** [comment]
This unfortunately was not reviewed and merged in time and is now in conflict with current structure or content.

Please create a new PR to apply these changes to the PDP docs and tag me as reviewer @ankur-arch @nurul3101 @luanvdw - thank you.

**Comment by nikolasburk** [comment]
Closing this PR because the page doesn't exist any more and there's no equivalent.

**Pull request closed without merging**